internal/cluster/distribution/eks/service.go

```diff
@@ -121,6 +123,7 @@ func NewNodePoolFromCFStack(name string, labels map[string]string, stack *cloudf
 		NodeSpotPrice               string `mapstructure:"NodeSpotPrice"`
 		NodeVolumeSize              int    `mapstructure:"NodeVolumeSize"`
 		Subnets                     string `mapstructure:"Subnets"`
+		CustomNodeSecurityGroups    string `mapstructure:"CustomNodeSecurityGroups,omitempty"`
```

Comment:
**Note**: listing requires other changes as well (and should be in a separate PR), I have submitted the corresponding change set [here](https://github.com/banzaicloud/pipeline/pull/3337).

Reply:
Same at https://github.com/banzaicloud/pipeline/pull/3328/files#diff-07fa2a5916b261c493d7d45fe3fc856a5203f30b8219443b62b5c08f41839382R99

Reply:
Fixed